cmd/noms/noms_serve.go

```diff
@@ -37,6 +37,7 @@ func setupServeFlags() *flag.FlagSet {
 	serveFlagSet.IntVar(&port, "port", 8000, "port to listen on for HTTP requests")
 	spec.RegisterDatabaseFlags(serveFlagSet)
 	verbose.RegisterVerboseFlags(serveFlagSet)
+	profile.RegisterProfileFlags(serveFlagSet)
```

Comment:
I wanted to be able to profile the noms server, and figured that's a thing we'd want to keep.